server/server.go

```diff
@@ -72,6 +72,7 @@ type Info struct {
 	MaxPayload        int32    `json:"max_payload"`
 	IP                string   `json:"ip,omitempty"`
 	CID               uint64   `json:"client_id,omitempty"`
+	ClientHost        string   `json:"client_ip,omitempty"`
```

Comment:
Since this is really going to be an IP, should it be `ClientIP` instead?

Reply:
sure. Will change before merge.